frontend/app/screens/welcome-screen/welcome-screen.tsx

```diff
@@ -37,7 +37,6 @@ const styles = StyleSheet.create({
     height: 75
   },
   button: {
-    width: 135,
```

Comment:
what was the problem?

Reply:
The e was on a new line for this button.